matrix/common/aws/dynamo_handler.py

```diff
@@ -88,7 +89,8 @@ def create_state_table_entry(self,
                 StateTableField.COMPLETED_QUERY_EXECUTIONS.value: 0,
                 StateTableField.EXPECTED_CONVERTER_EXECUTIONS.value: 1,
                 StateTableField.COMPLETED_CONVERTER_EXECUTIONS.value: 0,
-                StateTableField.CREATION_DATE.value: date.get_datetime_now(as_string=True)
+                StateTableField.CREATION_DATE.value: date.get_datetime_now(as_string=True),
+                StateTableField.BATCH_JOB_ID.value: "N/A"
```

Comment:
Consider storing "N/A" in `constants.py`

Reply:
Considered. Leaving as is